modules/nvidia_plugin/src/ops/subgraph.cpp

```diff
@@ -122,6 +122,23 @@ std::vector<DevicePointer<void*>> SubGraph::getSharedWorkbuffers(const IOperatio
     return result;
 }
 
+void SubGraph::Capture(InferenceRequestContext &context, Inputs inputTensors, Outputs outputTensors,
```

Comment:
They are unused in this implementation, but this is interface method for all operators. I think it will be useful for TensorIterator-like operators, therefore these arguments were kept in the interface.

Reply:
I removed these parameters to avoid warnings about unused variables.